google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/AssetSnippets.java

```diff
@@ -0,0 +1,171 @@
+    this.securityCenterClient = client;
+    // [START name_from_id]
+    this.organizationName = OrganizationName.of(organizationId);
+    // [END name_from_id]
+  }
+
+  /**
+   * Lists assets for an organization given meeting <code>filter</code> as of a specific instant in
+   * time.
+   *
+   * @param filter The filter that assets must meet (e.g. {@link #PROJECT_ASSET_FILTERS}). If null,
+   *     all assets in the organization are returned.
+   * @param asOf The instant in time to query for. If null, current time is assumed
+   */
+
+  // [TARGET list_assets]
+  // [VARIABLE "filter"]
+  // [VARIABLE "as_of"]
+  // [START list_assets]
+  public ImmutableList<ListAssetsResult> listAssets(String filter, Instant asOf) {
+    ListAssetsRequest.Builder request =
+        ListAssetsRequest.newBuilder().setParent(organizationName.toString());
+    // Default for API is to return all assets.
+    if (filter != null) {
+      request.setFilter(filter);
+    }
+    // Limits assets returned to a particular point in time.
+    if (asOf != null) {
+      request.getReadTimeBuilder().setSeconds(asOf.getEpochSecond()).setNanos(asOf.getNano());
+    }
+    ListAssetsPagedResponse response = securityCenterClient.listAssets(request.build());
+
+    // This creates one list for all assets.  If your organization has a large number of assets
+    // this can cause out of memory issues.  You can process them batches by returning
+    // the Iterable returned response.iterateAll() directly.
+    return ImmutableList.copyOf(response.iterateAll());
+  }
+  // [END list_assets]
+
+  /** Run and print results from common queries. */
+  void demoListAssets() {
+    // [START demo_list_assets]
+    // This takes care of formatting the resource name appropriately from the id..
+
+    // Query for all currently existing assets
+    System.out.println("All Assets: " + listAssets(null, null));
+    // Query for all projects as of now.
+    System.out.println("Project Assets (now): " + listAssets(PROJECT_ASSET_FILTERS, null));
+    // Query for all projects as of a day ago.
+    System.out.println(
+        "Project Assets (1 day ago): "
+            + listAssets(PROJECT_ASSET_FILTERS, Instant.now().minus(Duration.ofDays(1))));
+    // [END demo_list_assets]
+  }
+
+  /**
+   * Returns Assets and metadata about assets activity (e.g. added, removed, no change) between
+   * between <code>asOf.minus(timespan)</code> and <code>asOf</code>.
+   *
+   * @param timeSpan The time-range to compare assets over.
+   * @param filter The filter that assets must meet (e.g. {@link #PROJECT_ASSET_FILTERS}). If null,
+   *     all assets in the organization are returned.
+   * @param asOf The instant in time to query for. If null, current time is assumed.
+   */
+  public ImmutableList<ListAssetsResult> listAssetAndStatusChanges(
+      Duration timeSpan, String filter, Instant asOf) {
+    // [START list_asset_changes]
```

Comment:
Please add commented variables with a small description of what it's for?
Similar to: https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java#L1459-L1460

Reply:
I think I documented what you asked for.